QUEST_LV_0100.tsv

```diff
@@ -119,82 +119,82 @@ QUEST_LV_0100_20150317_000118	${memo X}Isn't it better to use the cable car rath
-QUEST_LV_0100_20150317_000163	If Jonas was walking with his maid, then that's after you went outside.{nl}Jonas was found when he was in his dotage and the maid is still missing.
-QUEST_LV_0100_20150317_000164	Please take good care of Jonas.{nl}It would be a big problem if he got hurt.
-QUEST_LV_0100_20150317_000165	Davio is at the right side of the camp.{nl}I hope the medicine works well.
-QUEST_LV_0100_20150317_000166	Finally, I remember everything.{nl}I am the head of the family that protects the Mausoleum... Who are you?{nl}
-QUEST_LV_0100_20150317_000167	Revelator. If this is the decision of the Goddess, then you must be right.{nl}You have a job to do, so follow me to the light signal place.
-QUEST_LV_0100_20150317_000168	I have made Niels and you worry...
-QUEST_LV_0100_20150317_000169	I will lead the way and you will handle any monsters we encounter.{nl}I am counting on you.
-QUEST_LV_0100_20150317_000170	Now is our chance. Follow me!
+QUEST_LV_0100_20150317_000142	$The Goddesses sent you?{nl}I never thought that you would protect me.{nl}
+QUEST_LV_0100_20150317_000143	$Where did that huge scorpion go? This is very bad!
+QUEST_LV_0100_20150317_000144	$I should go see Dezic.{nl}I hope to see you again in the future.
+QUEST_LV_0100_20150317_000145	$Someone must have done something strange to the monsters protecting the Royal Mausoleum.{nl}What were they thinking?
+QUEST_LV_0100_20150317_000146	$Sir, get a hold of yourself. Sir!
+QUEST_LV_0100_20150317_000147	$New Researcher
+QUEST_LV_0100_20150317_000148	$I lost my important research materials at the Abandoned Old Workshop.{nl}The monsters must have taken them... Should I just give up?
+QUEST_LV_0100_20150317_000149	$I am so glad I get to see the Royal Mausoleum myself, but the monsters... They are scary.
+QUEST_LV_0100_20150317_000150	$Thank you so much.{nl}But you should also check the Records Repository.
+QUEST_LV_0100_20150317_000151	$The entrance to the Royal Mausoleum is hidden within this gigantic canyon somewhere.{nl}The start of the canyon begins here.
+QUEST_LV_0100_20150317_000152	$Many experts have come here to conduct research on the Royal Mausoleum.{nl}Maybe we can find some clues about the Medis Diena here.
+QUEST_LV_0100_20150317_000153	$Why are there so many monsters in the canyon when there is so little plant life around?
+QUEST_LV_0100_20150317_000154	$What happened to my request? Is it not done yet?
+QUEST_LV_0100_20150317_000155	$Look closely at the ground.{nl}It may have fallen down.
+QUEST_LV_0100_20150317_000156	$Hmm, only these ones?{nl}Well, thank you for finding this much at least.{nl}I can just copy again from the beginning.
+QUEST_LV_0100_20150317_000157	$Niels sent you too?{nl}I am not going. I have to research the wild flowers.{nl}
+QUEST_LV_0100_20150317_000158	$I guess what I saw was an illusion.{nl}I must reward you for protecting a mineralogist like me.{nl}
+QUEST_LV_0100_20150317_000159	$Yes.{nl}Bring my archive from Niels.
```

Comment:
I don't think that's better. Record, or rather records, sounds more natural. Archive (singular) is usually the place, archives (plural) are usually the documents themselves. So either records, or archives, imo.